src/_update.sh

```diff
@@ -25,7 +25,7 @@ function mod_update() {
   # Sometimes containers won't start correctly at first, but will upon a retry
   maxRetries=2
   for i in $( seq 1 $maxRetries ); do
-    mod_start
+    mod_start || true
```

Comment:
Took me awhile to realize that _Docker Compose_ is throwing an error on `up` which is why the below checks aren't getting triggered in the particular case they're meant to handle. This is a mega simple change to keep from going into cleanup immediately, and instead retry the update.

Reply:
```suggestion
    mod_start || sleep 5 # Wait before going on to health checks, they should handle triggering retries if mod_start errors
```

Reply:
@mmorgan-pt thoughts?